src/shared/components/ConnectionList/ConnectionList.tsx

```diff
@@ -10,56 +11,64 @@ type ConnectionProps = {
 };
 
 type ConnectionListProps = {
-  theme: any;
   connectionList: ConnectionProps[];
 };
-const ConnectionList = ({ theme, connectionList }: ConnectionListProps) => {
+
+const ConnectionList = ({ connectionList }: ConnectionListProps) => {
+  const {
+    themeColor: { color, backgroundColor, navButtons },
+  } = useTheme();
   return (
     <Box
       sx={{
+        bgcolor: backgroundColor,
         width: "100%",
-        height: "100vh",
-        color: theme.color,
-        bgcolor: theme.backgroundColor,
-        display: "flex",
-        justifyContent: "center",
       }}
     >
       <Box
         sx={{
-          position: "fixed",
+          position: "sticky",
```

Comment:
do we want the list to stay up top or move down along with us as we scroll down?

Reply:
it stays right now